src/OpenSage.Game/Data/W3d/W3dTextureInfo.cs

```diff
@@ -33,7 +33,7 @@ internal static W3dTextureInfo Parse(BinaryReader reader, W3dParseContext contex
             {
                 return new W3dTextureInfo
                 {
-                    Attributes = reader.ReadUInt16AsEnumFlags<W3dTextureFlags>(),
+                    Attributes = (W3dTextureFlags) reader.ReadUInt16(),
```

Comment:
Why this change? I'd prefer we add the new values to `W3dTextureFlags`.

Reply:
Looks like I also changed this because it was failing to parse based on a new value.  I'll revert it and add as a UnknownENB value in the enum. Its value is: ... 0x0100;